modules/serverless-sinkbinding-kn.adoc

```diff
@@ -0,0 +1,138 @@
+----
+
+. Create a SinkBinding that will direct events to the service.
++
+You can create the SinkBinding by entering the following command:
++
+----
+$ kn source binding create bind-heartbeat --subject job:batch/v1:app=heartbeat-cron --sink svc:event-display
+----
+
+. Create a CronJob.
++
+You can do this by creating a file named `heartbeats-cronjob.yaml` and copying the following sample code into it:
++
+----
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: heartbeat-cron
+spec:
+spec:
+  # Run every minute
+  schedule: "* * * * *"
+  jobTemplate:
+    metadata:
+      labels:
+        app: heartbeat-cron
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: single-heartbeat
+              image: quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.13.2
+              args:
+                - --period=1
+              env:
+                - name: ONE_SHOT
+                  value: "true"
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+----
++
+After you have created the `heartbeats-cronjob.yaml` file, apply it by entering the following command:
++
+----
+$ oc apply --filename heartbeats-cronjob.yaml
+----
+
+. Check that the controller is mapped correctly by entering the following command and inspecting the output:
++
+----
+$ kn source binding describe bind-heartbeat
+----
++
+The output should be similar to:
++
+----
+Name:         bind-heartbeat
+Namespace:    demo-2
+Annotations:  sources.knative.dev/creator=minikube-user, sources.knative.dev/lastModifier=minikub ...
+Age:          2m
+Subject:
+  Resource:   job (batch/v1)
+  Selector:
+    app:      heartbeat-cron
+Sink:
+  Name:       event-display
+  Resource:   Service (serving.knative.dev/v1)
+
+Conditions:
+  OK TYPE     AGE REASON
+  ++ Ready     2m
+----
+
+.Verification steps
+
+You can verify that the Kubernetes events were sent to the Knative event sink by looking at the message dumper function logs.
+
+You can view the message dumper function logs by entering the following commands:
+----
+$ oc get pods
+$ oc logs $(oc get pod -o name | grep event-display) -c user-container
+----
+
+The logs should contain lines similar to the following:
+----
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.eventing.samples.heartbeat
+  source: https://knative.dev/eventing-contrib/cmd/heartbeats/#event-test/mypod
+  id: 2b72d7bf-c38f-4a98-a433-608fbcdd2596
+  time: 2019-10-18T15:23:20.809775386Z
+  contenttype: application/json
+Extensions,
+  beats: true
+  heart: yes
+  sink: bound
+  the: 42
+Data,
+  {
+    "id": 2,
```

Comment:
should be "id": 1 

it can only be 1, as the CronJob repeatedly deploys the heartbeats container with ONE_SHOT=true  , so it will produce a sequence of one-shot heartbeats, so the id will always be just 1